ansible/playbook/tasks/roles/nginx/files/conf.d/pmm.conf

```diff
@@ -111,6 +121,11 @@
       proxy_read_timeout 600;
     }
 
+    location /prometheus/api/v1 {
```

Comment:
shouldn't we remove prometheus location above?

Reply:
no, since this particular location protects access to metrics itself, but `/prometheus` should be kept untouched in order to keep Victoria UI (images, css, etc) working